app.rb

```diff
@@ -71,7 +71,7 @@
 #  db.execute("INSERT INTO shopping (id, item, quantity, store, section) VALUES #{params['id']} #{params['item']}
   # @test = db.execute("INSERT INTO shopping (item, quantity, store, section) VALUES (#{params['item']},  #{params['quantity']}, #{params['store']}, #{params['section']})")
   # query the products table and print the result
-  db.execute("INSERT INTO shopping (item, quantity, store, section) VALUES ('#{params['item']}', '#{params['quantity']}', '#{params['store']}', '#{params['section']}')")
+  # db.execute("INSERT INTO shopping (item, quantity, store, section) VALUES ('#{params['item']}', '#{params['quantity']}', '#{params['store']}', '#{params['section']}')")
```

Comment:
This should not be commented out.

Reply:
Whooooopsies. Sorry about that. I'll fix